File: B/C++/TimSepakBola.cpp
class TimSepakBola {
	private:
		string namaTim;		// Atribut nama tim
		string negara;		// Atribut asal negara
		int tahun;		// Atribut tahun berdiri
		string namaPemain[20];	// Atribut nama pemain

	public:
		// Kosntruktor
		TimSepakBola() {}

		/* Set & Get */
		
		// Nama Tim
		void setNamaTim(string namaTim) {
			this->namaTim = namaTim;
		}
		string getNamaTim() {
			return this->namaTim;
		}

		// Asal Negara Tim
		void setNegara(string negara) {
			this->negara = negara;
		}
		string getNegara() {
			return this->negara;
		}

		// Tahun berdiri Tim
		void setTahun(int tahun) {
			this->tahun = tahun;
		}
		int getTahun() {
			return this->tahun;
		}

		// Nama Pemain
		void setNamaPemain(string namaPemain[], int jumlahPemain) {
			for(int i = 0; i < jumlahPemain; i++) {
				this->namaPemain[i] = namaPemain[i];
			}
		}
		string getNamaPemain(int jumlahPemain) {
			return this->namaPemain[jumlahPemain];
		}

		// Destruktor
		~TimSepakBola() {}
};

File: B/Python/TimSepakBola.py
class TimSepakBola:

	# Konstruktor
	def __init__(self, jumlahPemain):
		self.namaTim = ""
		self.negara = ""
		self.tahun = 0
		self.namaPemain = ["0"]*jumlahPemain

	# Set & Get
	
	# Nama Tim
	def setNamaTim(self, namaTim):
		self.namaTim = namaTim

	def getNamaTim(self):
		return self.namaTim

	# Asal Negara
	def setNegara(self, negara):
		self.negara = negara

	def getNegara(self):
		return self.negara

	# Tahun Berdiri
	def setTahun(self, tahun):
		self.tahun = tahun

	def getTahun(self):
		return self.tahun

	# Nama
	def setNamaPemain(self, namaPemain, jumlahPemain):
		for i in range(jumlahPemain):
			self.namaPemain[i] = namaPemain[i]

	def getNamaPemain(self, jumlahPemain):
		return self.namaPemain[jumlahPemain]